appveyor.yml

```diff
@@ -43,12 +46,12 @@ environment:
   # Stable 32-bit MSVC
     - channel: stable
       target: i686-pc-windows-msvc
-  # Beta 64-bit MSVC
-    - channel: beta
-      target: x86_64-pc-windows-msvc
-  # Beta 32-bit MSVC
-    - channel: beta
-      target: i686-pc-windows-msvc
+#  # Beta 64-bit MSVC
+#    - channel: beta
+#      target: x86_64-pc-windows-msvc
+#  # Beta 32-bit MSVC
```

Comment:
Should this comments be removed?

Reply:
Actually I plan to use this properties in nearest future. So let it be 👌 ?